src/themes/default/App.vue

```diff
@@ -71,7 +71,8 @@ export default {
 </script>
 
 <!-- CSS DOCS: https://github.com/DivanteLtd/vue-storefront/blob/master/doc/Working%20with%20stylesheets%20(CSS).md -->
-<style src="./css/flexbox-grid.css"></style>
+<style src="../../../node_modules/flexboxgrid2/flexboxgrid2.min.css"></style>
```

Comment:
We should have used the web pack alias in that case or other way to keep the css unaware of physical path structure @pakiam @filrak @mercs600

Reply:
@pkarw added to vue-loader.config option IncludePath: '/node_modules/'